pkgs/development/libraries/libwebp/default.nix

```
@@ -32,7 +32,7 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner  = "webmproject";
     repo   = pname;
-    rev    = version;
+    rev    = "v" + version;
```

most common way to express this
```suggestion
    rev    = "v${version}";
```